framework/src/userobject/ThreadedRadialAverageLoop.C

```diff
@@ -50,7 +50,7 @@ ThreadedRadialAverageLoop::operator()(const QPDataRange & qpdata_range)
 
     ret_matches.clear();
     std::size_t n_result =
-        kd_tree->radiusSearch(&(local_qp._q_point(0)), radius, ret_matches, search_params);
+        kd_tree->radiusSearch(&(local_qp._q_point(0)), radius * radius, ret_matches, search_params);
```

Comment:
What's this fix about?

Reply:
That's how nanoflann expects the radius with the default L2 metric